src/parser/esprima_cpp/esprima.cpp

```diff
@@ -4433,7 +4486,9 @@ class Parser : public gc {
                 }
             } else if (this->matchKeyword(ConstKeyword) || this->matchKeyword(LetKeyword)) {
                 // TODO
-                this->throwUnexpectedToken(this->nextToken());
+                Scanner::ScannerResult* token = STACK_ALLOCA(sizeof(Scanner::ScannerResult), Scanner::ScannerResult);
+                this->nextToken(token);
+                this->throwUnexpectedToken(token);
                 /*
                 init = this->createNode();
```

Comment:
remove if not active codes

Reply:
Our parser is based on the esprima parser. This comment is related with esprima code which is left to implement es6 features later.